### PR TITLE
goreadline/minimega: sigint like bash

### PR DESCRIPTION
**Pull request by jcrussell**
Respond to Ctrl-C like bash: kill the current line and start with a new prompt.

Only thing that is different is that the reverse history line gets cleared when searching is aborted by Ctrl-C.

### CODE REVIEW

File: src/goreadline/goreadline.c [hunk 1]
```diff
@@ -1,11 +1,85 @@
-// Copyright (2015) Sandia Corporation.
```
Comment (djfritz) [inline src/goreadline/goreadline.c]: Why did we lose the copyright header?

File: src/goreadline/goreadline.go [hunk 2]
```diff
@@ -84,13 +91,13 @@ func minicliCompletion(text *C.char, start, end C.int) **C.char {
 // Rlwrap prompts the user with the given prompt string and calls the
 // underlying readline function. If the input stream closes, Rlwrap returns an
 // EOF error.
-func Rlwrap(prompt string, record bool) (string, error) {
+func Readline(prompt string, record bool) (string, error) {
```
Comment (djfritz) [inline src/goreadline/goreadline.go]: Function name changed, but function name in the comment didn't.

File: src/goreadline/goreadline.c (hunk 1 above)
Comment (jcrussell) [inline src/goreadline/goreadline.c]: Whoops, deleted that whole file and forgot to re-add the header.

File: src/goreadline/goreadline.c [hunk 3]
```diff
@@ -1,11 +1,87 @@
...
-// Under the terms of Contract DE-AC04-94AL85000 with Sandia Corporation,
```
Comment (djfritz) [inline src/goreadline/goreadline.c]: What about the rest of the header?

**Comment by djfritz** [comment]
LGTM

**Pull request merged**